tests/ui/specs/Scroller/EditableScrollerItem/EditableScrollerItem-specs.js

```diff
@@ -0,0 +1,195 @@
+const ScrollerPage = require('./EditableScrollerItemPage');
+const {expectFocusedItem} = require('../Scroller-utils');
+
+describe('Editable Scroller Item', function () {
+	beforeEach(async function () {
+		await ScrollerPage.open();
+		await ScrollerPage.spotlightDown();
+		await ScrollerPage.buttonEditMode.isFocused();
+		await ScrollerPage.spotlightSelect();
+		await ScrollerPage.spotlightDown();
+		await expectFocusedItem(0);
+	});
+
+	it('Should remove item with remove button onSelect', async function () {
```

Comment:
each test case description should begin with a lower case. I don't know why `EditableScroller` has them with upper case, but we need to align them with the other. please modify each test case to begin with a lower case letter